src/utils/form/getDisplayText.ts

```diff
@@ -0,0 +1,52 @@
+import { CustomAPI } from '../../server/apollo/dataSources';
+
+/**
+ * Gets display text from choice value.
+ * @param choices list of choices.
+ * @param value choice value.
+ * @returns display value of the value.
+ */
+const getText = (choices: any[], value: any): string => {
```

Comment:
@PacomeRivier, always put the /** comments, so we can be sure to never leave a method without explanation